.github/workflows/nightly.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 permissions:
   contents: read
+  pull-requests: write
```

Comment:
you can set this extra permissions only for `Benchmarks` job below

see e.g. https://github.com/oneapi-src/unified-memory-framework/blob/main/.github/workflows/pr_push.yml#L81